docroot/modules/custom/mass_feedback_loop/src/Form/MassFeedbackLoopAuthorInterfaceForm.php

```diff
@@ -295,6 +285,17 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#prefix' => '<div id="table-wrapper">',
       '#suffix' => '</div>',
     ];
+    // Attaches necessary JS library to run single-page app.
+    $form['#attached']['library'][] = 'mass_feedback_loop/mass-feedback-author-interface';
+
+    // Adds sorting information to drupalSettings.
+    $form['#attached']['drupalSettings']['massFeedbackLoop']['sortingVariants'] = MassFeedbackLoopContentFetcher::SORTING_VARIANTS;
+
+    // Early return if form has not yet been submitted.
+    if (!\Drupal::request()->getQueryString()) {
```

Comment:
This is what avoids doing a query when no input has been provided. The rest of the PR is changing the default value of watch_content